pykafka/pykafka/simpleconsumer.py

```diff
@@ -292,9 +310,8 @@ def _reset_offsets(self, errored_partitions):
             reqs = [owned_partition.build_offset_request(self._auto_offset_reset)
```

Comment:
[nitpick] from a few lines above `owned_partitions_by_leader` is quite a mouthful.  It's not a long-lived variable so something simple like `by_partition` shortens the lines before and makes them easier to read through.